apps/site/assets/ts/models/alert.ts

```diff
@@ -17,6 +17,15 @@ export const isDiversion = ({ effect }: Alert): boolean =>
 export const isHighPriorityAlert = ({ effect }: Alert): boolean =>
   effect === "detour" || effect === "suspension" || effect === "shuttle";
 
+export const alertsForEffects = (alerts: Alert[], effects: string[]): Alert[] =>
+  alerts.filter(a => effects.includes(a.effect));
+
+export const alertsForEffect = (alerts: Alert[], effect: string): Alert[] =>
```

Comment:
questions: Is alertsForEffect being used anywhere? Is there a reason `alertsForEffects` can't be called directly rather than using `alertsForEffect` and `filterParkingAlerts`?